contracts/lib/Structs.sol

```diff
@@ -3,21 +3,23 @@ pragma solidity ^0.8.0;
 
 library Structs {
     struct Vote {
-        uint256 value;
-        uint256 weight;
+        uint32 epoch;
+        uint48[] values;
     }
 
+    struct Commitment {
+        uint32 epoch;
+        bytes32 commitmentHash;
+    }
     struct Staker {
-        uint256 id;
-        address _address;
-        uint256 stake;
-        uint256 age;
-        uint256 epochStaked;
-        uint256 epochLastCommitted;
-        uint256 epochLastRevealed;
         bool acceptDelegation;
-        uint256 commission;
+        uint32 id;
+        uint32 age;
+        uint32 epochStaked;
+        address _address;
         address tokenAddress;
+        uint256 commission;
```

Comment:
we can make commission uint8 as its always <=100 and move it to top

Reply:
done

Reply:
done